fava/application.py

```diff
@@ -124,6 +125,21 @@ def _inject_filters(endpoint, values):
             values[filter_name] = g.filters[filter_name]
 
 
+@app.url_defaults
+def _static_modified(endpoint, values):
+    """Add a query param to force cache updates when static files change."""
+    if endpoint != 'static':
+        return
+
+    filename = values.get('filename', None)
+    if not filename:
+        return
+
+    file_path = os.path.join(app.static_folder, filename)
+    if os.path.exists(file_path):
+        values['_'] = int(os.stat(file_path).st_mtime)
```

Comment:
maybe replace `_` by `mtime` - no need to be cryptic here

Reply:
Done